app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java

```diff
@@ -143,11 +138,42 @@ public void onClick(View v) {
                 }
             }
         });
+        ToggleButton toggleButton = (ToggleButton) view.findViewById(R.id.toggle_override);
+        toggleButton.setTextOn(getResources().getString(R.string.off));
+        toggleButton.setTextOn(getResources().getString(R.string.override));
```

Comment:
Also, don't set it programatically but in XML with android:textOn and android:textOff.